### PR TITLE
Navigation top bar

### PR DESCRIPTION
**Pull request by BastienSaulnier**
New TopBar system for website navigation.

### CODE REVIEW

**Review by LeoDiduch — APPROVED** [review]
OK

File: frontend/src/components/NavBar.jsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+          </li>
+
+          <li>
+            <NavLink
+              id="meetings"
+              className="menu-item"
+              activeClassName="active"
+              to="/ambassadors"
+            >
+              Ambassadors
```
Comment (LoicBrassart) [inline frontend/src/components/NavBar.jsx:34]: Ambassadeurs

File: frontend/src/components/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+          Rencontres
+        </NavLink>
+
+        <NavLink
+          id="meetings"
+          className="menu-item"
+          activeClassName="bm-active"
+          to="/ambassadors"
+        >
+          Ambassadors
```
Comment (LoicBrassart) [inline frontend/src/components/NavBar.jsx:78]: Là aussi.
... Du moins je crois ? Ca me parait plus logique en tous cas

**Review by LoicBrassart — APPROVED** [review]
Ok for me
There is (probalby) a typo, I'll let you choose if you want to correct it now or later

**Pull request merged**